output.json

```diff
@@ -0,0 +1,3 @@
+{"type": "STATE", "value": {"currently_syncing": "bank_transactions", "bookmarks": {"bank_transactions": {"UpdatedDateUTC": "2000-01-01", "offset": {"page": 1}}}}}
```

Comment:
not sure how this is include as it's supposed to be excluded by `.gitignore`

Reply:
yeah I added the ignore line after. Will delete